love_letter_sim2.py

```diff
@@ -130,15 +131,18 @@ def more_than_one_player(self):
 
     def take_turn(self):
         player = next(self.turn_order)
+        player.protected = False # effect of a previously played Handmaid expires
```

Comment:
Aha!  This fixes the problem I was trying to work around.  Didn't think to put that here.